js/field.js

```diff
@@ -0,0 +1,228 @@
+function setOnClicksAndOtherThings() {
+    
+}
+
+var field = document.getElementById('field');
+var parentCard;
+var tempCard;
+var totalCountInAcePiles=0;
+field.onmousedown = onMouseDown;
+
```

Comment:
Why don't you use oop? I think it would be more comprehensive and understandable to create field like component (instance of some class) and add different methods.